README.adoc

```diff
@@ -566,6 +566,23 @@ data:
 
 == Additional Notes
 
+=== Customize the UI web context root
+
+By default Kiali UI is deployed to the top level of `https://kiali-istio-system.<your_cluster_domain_or_ip>/`.  In some situation such as when you want to serve Kiali UI along with other apps under the same host name, e.g., `example.com/kiali`, `example.com/app1`, you can edit Kiali Config Map and provide a different value for `web_root`.  Note: the path must begin with a `/`.
+
+An example of custom web root:
+
+[source,yaml]
+----
+server:
+  port: 20001
+  static_content_root_directory: /opt/kiali/console
```

Comment:
You can delete this line - another PR that is about to be merged is going to remove this.

Reply:
I am referring to this: https://github.com/kiali/kiali/pull/515/files